test/mincost.jl

```diff
@@ -14,83 +16,67 @@ using SparseArrays: spzeros
     lg.add_edge!(g, 1, 4)
     lg.add_edge!(g, 2, 3)
     lg.add_edge!(g, 2, 4)
-    w = zeros(6,6)
-    w[1,3] = 10.
-    w[1,4] = 5.
-    w[2,3] = 2.
-    w[2,4] = 2.
+    cost = zeros(6,6)
+    cost[1,3] = 10.
+    cost[1,4] = 5.
+    cost[2,3] = 2.
+    cost[2,4] = 2.
     # v2 -> sink have demand of one
     demand = spzeros(6,6)
     demand[3,6] = 1
     demand[4,6] = 1
     capacity = ones(6,6)
 
-    flow = mincost_flow(g, spzeros(lg.nv(g)),capacity, w, ClpSolver(), edge_demand=demand,source_nodes=[5],sink_nodes= [6])
-    @test flow[5,1] == 1
-    @test flow[5,2] == 1
-    @test flow[3,6] == 1
-    @test flow[4,6] == 1
-    @test flow[1,4] == 1
-    @test flow[2,4] == 0
-    @test flow[2,3] == 1
-    @test flow[1,3] == 0
-    @test sum(diag(flow)) == 0
+    o = with_optimizer(Clp.Optimizer)
```

Comment:
This was deprecated in JuMP 0.21 I think; but in JuMP 0.21 I believe you can just pass `Clp.Optimizer` itself.